From d77a39f3f09993fe33ecd7bf75a41018ca982a8f Mon Sep 17 00:00:00 2001
From: Allan Anderson <agander93@gmail.com>
Date: Fri, 29 Mar 2013 22:02:52 +0000
Subject: [PATCH 22/22] BUG:316042 - KMyMoney hangs when editing transaction in
 anon file. Fix problem when creating an Anon file, where a priceMode setting
 in an investment or a stock account, was anonymized, resulting in looping in
 InvestTransactionEditor::priceMode(void) on attempting to edit the account in
 question. (cherry picked from commit
 240289fa0c3391dd71d0c13a44a43fa6fd8deaf7)

---
 kmymoney/mymoney/storage/mymoneystorageanon.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kmymoney/mymoney/storage/mymoneystorageanon.cpp b/kmymoney/mymoney/storage/mymoneystorageanon.cpp
index a2c626e..da600a6 100644
--- a/kmymoney/mymoney/storage/mymoneystorageanon.cpp
+++ b/kmymoney/mymoney/storage/mymoneystorageanon.cpp
@@ -43,7 +43,7 @@
 #include "mymoneyreport.h"
 #include "mymoneyinstitution.h"
 
-QStringList MyMoneyStorageANON::zKvpNoModify = QString("kmm-baseCurrency,PreferredAccount,Tax,fixed-interest,interest-calculation,payee,schedule,term,kmm-online-source,kmm-brokerage-account,lastStatementDate,kmm-sort-reconcile,kmm-sort-std,kmm-iconpos,mm-closed,payee,schedule,term,lastImportedTransactionDate,VatAccount,VatRate,kmm-matched-tx,Imported").split(',');
+QStringList MyMoneyStorageANON::zKvpNoModify = QString("kmm-baseCurrency,PreferredAccount,Tax,fixed-interest,interest-calculation,payee,schedule,term,kmm-online-source,kmm-brokerage-account,lastStatementDate,kmm-sort-reconcile,kmm-sort-std,kmm-iconpos,mm-closed,payee,schedule,term,lastImportedTransactionDate,VatAccount,VatRate,kmm-matched-tx,Imported,priceMode").split(',');
 QStringList MyMoneyStorageANON::zKvpXNumber = QString("final-payment,loan-amount,periodic-payment,lastStatementBalance").split(',');
 
 
-- 
1.8.1.4

